src/components/Log/LogDateItem.tsx

```diff
@@ -0,0 +1,23 @@
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import DateRangeIcon from '@mui/icons-material/DateRange';
+import { Box, Typography } from '@mui/material';
+
+interface LogDateItemProps {
+  date: Date | string | number;
+}
+
+export const LogDateItem = ({ date }: LogDateItemProps) => {
+  const dateObject = date instanceof Date ? date : new Date(date);
+
+  const dateString = dateObject.toLocaleDateString('nb-NO', { year: 'numeric', month: '2-digit', day: '2-digit' });
+  const timeString = dateObject.toLocaleTimeString('nb-NO', { hour: '2-digit', minute: '2-digit' });
```

Comment:
Ser ut som vi bare sender inn `Date` her, og at dette dermed kan forenkles
```suggestion
  date: Date;
}

export const LogDateItem = ({ date }: LogDateItemProps) => {
  const dateString = date.toLocaleDateString('nb-NO', { year: 'numeric', month: '2-digit', day: '2-digit' });
  const timeString = date.toLocaleTimeString('nb-NO', { hour: '2-digit', minute: '2-digit' });
```